demos/python_demos/machine_translation_demo/machine_translation_demo.py

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/env python3
```

Comment:
Please, set linux execution permissions for the file

Reply:
done